src/data/teams.json

```diff
@@ -711,6 +711,24 @@
       }
     ]
   },
+  {
+    "title": "Research",
+    "contributors": [
+      {
+        "title": "Blockchain Researcher",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-11-19T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": null,
```

Comment:
Please add a `contributorId` here. Check the `contributorId`s in the development branch and choose the next in order.

For ex - if the highest `contributorId` right now is 55, use 56 here.